static/test/javascripts/spec/membership/payment-form.spec.js

```diff
@@ -61,8 +56,8 @@ define([
 
         beforeEach(function () {
             paymentFormFixtureElement = $.create(paymentFormHtml)[0];
-            paymentForm = new PaymentForm(guConfig);
-            paymentForm.init(paymentFormFixtureElement);
+            paymentForm = new PaymentForm(paymentFormFixtureElement, $(), '');
```

Comment:
hmm it turns out it is useful to be able to pass in DOM elements in the test to avoid having to modify the actual page we're testing on, how would you feel about specifically supporting passing either dom elements or css selectors to the payment form constructor?